site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -45,7 +45,7 @@ This is an optional list of domains that this SAML v2 provider should be used fo
 endif::[]
 
 [field]#identityProvider.emailClaim# [type]#[String]# [optional]#Optional#::
-The name of the email claim (Attribute in the `Assertion` element) in the SAML response that FusionAuth uses to uniquely identity the user. If this is not set, the `useNameIdForEmail` flag must be true.
+The name of the email claim in the SAML response that FusionAuth uses to uniquely identity the user. If this is not set, the `useNameIdForEmail` flag must be true.
```

Comment:
When we reference other fields such as `useNameIdForEmail` we should use the field syntax. `[field]#identityProvider.useNameIdForEmail#. 

The general convention for these optionally required fields is to use text such as:

> Required when [field]#identityProvider.useNameIdForEmail# is `false`. 

Search the docs for examples, we usually say this on a new line.